src/sms-password/templates/user_management/sms-password/reset_form.html

```diff
@@ -4,6 +4,7 @@
 
 <div class="alert alert-success" ng-show="changed">
     <p translate>You've successfully changed your password.</p>
+    <p translate>Please <a href="{{ 'HomeCtrl'|reverseUrl }}">log in</a> here</p>
```

Comment:
Well either override the template in the project or make a `$scope` variable that you can set in `config` ?